dualscreeninfo/src/twopaneview/TwoPaneView.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import { RNHinge as Hinge } from './RNHinge'
+
+export interface TwoPaneViewProps {
+  leftComponent: React.Component,
+  rightComponent: React.Component,
+}
+
+export const RNTwoPaneView = ({ leftComponent, rightComponent}: TwoPaneViewProps) => {
```

Comment:
>RNTwoPaneView [](start = 13, length = 13)

Alexandre Zollinger Chohfi has a good head-start on a TwoPaneView implementation, based on an early prototype of mine.  I suggest sync'ing with him to leverage what's already been done.

Reply:
that's only stub ;)

Reply:
Based on our conversation on Friday, Homero is going to work with Alex to incorporate TwoPaneView.  You can leave the stub in place for now.

---
In reply to: [382726663](https://github.com/microsoft/react-native-dualscreen/pull/2#discussion_r382726663) [](ancestors = 382726663)